package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "remimi",
-  "version": "1.0.0",
+  "version": "1.3.0",
```

Comment:
We could move to `semantic-release` it's working quite well over on [BrandwatchLtd/Axiom](https://github.com/BrandwatchLtd/axiom). What do you think @kamilio ?